devel/Dockerfile

```diff
@@ -44,6 +47,15 @@ RUN git clone --recursive https://github.com/phoenix-rtos/phoenix-rtos-project.g
     cd .. && rm -rf phoenix-rtos-project && \
     chmod -R a+rwX /opt/phoenix/utils
 
+# install openocd in 0.11 version
+RUN mkdir -p /opt/openocd/ && mkdir -p /etc/udev/rules.d && \
```

Comment:
- no need to build it in `/opt/openocd`
- please remove the downloaded source code after installing openocd
- please use `60-openocd.rules` file from the cloned repo (in `contrib` subdir)

Reply:
Thanks, it's updated now.